test/Microsoft.ML.CpuMath.PerformanceTests/PerformanceTests.cs

```diff
@@ -15,12 +15,18 @@ public abstract class PerformanceTests
         private const int ExponentMin = 0;
         private const int ExponentRange = ExponentMax / 8;
 
-        protected const int IndexLength = 1000003;
-        protected const int Length = 1000003;
-        protected const int MatrixIndexLength = 100;
+        protected const int IndexLength = 1000000;
```

Comment:
nit: Might it be good to ensure this continues working even with inputs that have odd lengths?